src/viur/core/modules/file.py

```diff
@@ -969,6 +1147,26 @@ def mark_for_deletion(self, dlkey: str) -> None:
 
         db.Put(fileObj)
 
+    def create_serving_url(self, skel: SkeletonInstance) -> SkeletonInstance:
+        """ Create Serving url for public image files
+        """
+        # try to create a servingurl for images
+        if not conf.instance.is_dev_server and skel["public"] and skel["mimetype"] \
+                and skel["mimetype"].startswith("image/") and not skel["serving_url"]:
+
+            try:
+                bucket = File.get_bucket(skel['dlkey'])
+                skel["serving_url"] = images.get_serving_url(
+                    None,
+                    secure_url=True,
+                    filename=f"/gs/{bucket.name}/{skel['dlkey']}/source/{skel['name']}",
+                )
+            except Exception as e:
+                logging.warning("Error while creating serving url")
+                if not conf.instance.is_dev_server:
+                    logging.exception(e)
```

Comment:
```suggestion
                logging.exception(e)
```
We don't need this check I think. Because we check this above.